src/Grid/grid.jl

```diff
@@ -184,11 +216,15 @@ function _face_neighbor!(V_face, I_face, J_face, cellid, cell, neighbor, neighbo
         face_neighbor = EdgeIndex((neighborid, neighbor_face_id))
     end
     cell_face_id = findfirst(x->issubset(x,neighbor_face),faces(cell))
-    push!(V_face, Neighbor(face_neighbor))
+    push!(V_face, EntityNeighborhood(face_neighbor))
     push!(I_face, cellid)
     push!(J_face, cell_face_id)
 end
 
+getelement(neighbor::EntityNeighborhood{T}) where T <: BoundaryIndex = first.(neighbor.neighbor_info)
+getelement(neighbor::EntityNeighborhood{CellIndex}) = getproperty.(neighbor.neighbor_info, :idx)
+getelements(neighbors::Vector{T}) where T <: EntityNeighborhood = reduce(vcat, getelement.(neighbors))
```

Comment:
While I personally also prefer the name `element` I am not sure if it is good in Ferrite, because we use this name almost nowhere. To be specific, now we have `cell`, `entity` and `element` refering to the same concept (, sometimes).